src/AnnouncementBubble/QuotedAnnouncementBubble.less

```diff
@@ -78,6 +78,36 @@
   margin-left: 0.3125rem;
 }
 
+.QuotedAnnouncementBubble--attachmentContainer {
+  max-width: 100%;
+  .margin--bottom--xs();
+}
+
+.MessagingBubble--Message
+  .QuotedAnnouncementBubble--attachmentContainer
+  .MessagingAttachment--Container {
+  width: 14rem;
+  .text--truncate();
+}
+
+// the below styles disable the MessagingAttachment's normal hover styles
+// for all of our usecases.
+.QuotedAnnouncementBubble--attachmentContainer--dark .MessagingAttachment--Container:hover {
+  background-color: #626776; /* stylelint-disable-line */
+  border-color: @neutral_silver;
+}
+
+.QuotedAnnouncementBubble--attachmentContainer--dark
+  .MessagingAttachment--Container:hover
+  .MessagingAttachment--Title,
+.QuotedAnnouncementBubble--attachmentContainer--dark .MessagingAttachment--TextContainer {
+  color: @neutral_white;
+}
+
+.QuotedAnnouncementBubble--attachmentContainer--dark img {
+  filter: brightness(0) invert(1); /* makes the file icon white without needing a different <img> src attr */
```

Comment:
capital 'F' Fancy 🧐

Reply:
Ooh, fancy indeed! Just to be certain, are we sure of cross-browser compatibility across Chrome, Safari, Firefox, and Edge? 😮

Reply:
Looks good for all of the above! https://caniuse.com/css-filters